tests/reducers/draftLabOrderReducer.test.js

```diff
@@ -1,11 +1,12 @@
 import draftLabOrderReducer from '../../app/js/reducers/draftLabOrderReducer';
 import {
   ADD_DRAFT_LAB_ORDER,
-  DELETE_DRAFT_LAB_ORDER
+  DELETE_DRAFT_LAB_ORDER,
+  TOGGLE_DRAFT_LAB_ORDER_URGENCY
 } from '../../app/js/actions/actionTypes';
 
 
-describe('Draft Lab Order Reducer', () => {
+describe.only('Draft Lab Order Reducer', () => {
```

Comment:
Doesn't this `only` means it will run only this describe block?

Reply:
Oh, was a mistake would fix this, I also noticed the CI is failing. I'll look into this

Reply:
Fixed